File: Python/A_NivelIniciante/Exercicio_1015.py
# -*- coding: cp1252 -*-
# -*- encoding: utf-8 -*-
#url do problema - https://www.urionlinejudge.com.br/judge/pt/problems/view/1015
linha1 = input().split(" ")
linha2 = input().split(" ")
x1, y1 = linha1
x2, y2 = linha2
resultado = (float(x2)-float(x1))**2 + (float(y2)-float(y1))**2
print("%.4f" %(resultado**(1/2)))

File: Javascript/A_NivelIniciante/Exercicio_1015.js
var input = require("./readFile.js").input;
var lines = input.split('\n');
var linha1 = lines.shift().split(" ");
x1 = linha1[0]
y1 = linha1[1]
var linha2 = lines.shift().split(" ");
x2 = linha2[0]
y2 = linha2[1]
var resultado = (parseFloat(x2)-parseFloat(x1))**2 + (parseFloat(y2)-parseFloat(y1))**2
console.log((resultado**(1/2)).toFixed(4));
